exercises/service-invocation/test/invoker_service_test.bal

```diff
@@ -8,6 +8,18 @@ function testFunc() {
     var response = httpEndpoint->get("/quote");
     if (response is http:Response) {
         test:assertEquals(response.statusCode, 200);
+        var payload = response.getTextPayload();
+        if (payload is string) {
+            boolean quoteFound = false;
+            foreach string quote in quotes {
+                if (quote == payload) {
+                    quoteFound = true;
+                }
+            }
```

Comment:
Would it be better if we use an iterable operation here?
Something like below?
```ballerina
array.foreach(function (string s) {
            if (s == "bc") {
                x = true;
            }
        });
```
It doesn't reduce the lines of code that much though.